docs/index.md

```diff
@@ -15,30 +15,29 @@ We answer the most common questions about contributing [in our contributor FAQ](
 
 ## Curriculum
 
-Our curriculum is curated by the global freeCodeCamp community. This way, we are able to incorporate expert knowledge from volunteers like you.
+Our curriculum is curated by the global freeCodeCamp community. This way, we are able to incorporate expert knowledge from volunteers like you. You can help by:
+- Expanding and improving the curriculum
+- Updating project user stories to better-explain concepts
+- Improving our automated tests to test people's code more accurately
 
-You can help expand and improve the curriculum. You can also update project user stories to better-explain concepts. And you can improve our automated tests so that we can more accurately test people's code.
-
-**If you're interested in improving our curriculum, here's [how to contribute to the curriculum](how-to-work-on-coding-challenges.md).**
+**If you're interested in improving our curriculum, here's how: [How do I contribute to the curriculum?](how-to-work-on-coding-challenges.md).**
```

Comment:
```suggestion
**If you're interested in improving our curriculum, here's how: [How do I contribute to the curriculum?](how-to-work-on-coding-challenges.md)**
```
Hyperlink text ends with a `?` so no need to append a `.` 🙂 

I'm not 100% sure on the verbiage here, though.